metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -419,6 +419,7 @@ This package installs the Metron Rest %{metron_home}
 %dir %{metron_home}/lib
 %{metron_home}/config/rest_application.yml
 %{metron_home}/bin/metron-rest.sh
+%{metron_home}/bin/pcap_to_pdml.sh
```

Comment:
Is there anything to be done for the ubuntu debs?

Reply:
I tested this by building the ubuntu debs with `mvn clean package -Pbuild-debs`.  I did not encounter any issues.